internal/configs/action.go

```diff
@@ -57,15 +57,28 @@ const (
 
 // ActionRef represents a reference to a configured Action
 type ActionRef struct {
-	Traversal hcl.Traversal
```

Comment:
🤔  I used a traversal here instead of an addr because this is used in parsing configs, where we can't always parse references on the first pass - how is that going to work now?

Reply:
Ah I wasn't aware that this is how it's divided. No I think I can also do the address parsing at a later point, that should be no problem. I think this is mostly me trying to figure things out